api/openapi/invitations.yml

```diff
@@ -104,7 +104,7 @@ paths:
       requestBody:
         $ref: "#/components/requestBodies/AcceptInvitationReq"
       responses:
-        "200":
+        "204":
```

Comment:
Please double-check. Accept seems to return 200 (https://github.com/SammyOina/mainflux/blob/556ffc1aa00a9bd40c477ae7086693717994f6ef/invitations/api/responses.go#L73). Either revert this or update the response to 204. While 200 is more common, 204 is the correct code here and I don't mind using it.